sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClientOptions.cs

```diff
@@ -56,6 +57,34 @@ public class ServiceBusClientOptions
         ///
         public Uri CustomEndpointAddress { get; set; }
 
+        /// <summary>
+        ///   The amount of time to allow a connection to have no observed traffic before considering
+        ///   it idle and eligible to close.
+        /// </summary>
+        ///
+        /// <value>The default idle timeout is 60 seconds.  The timeout must be a positive value.</value>
+        ///
+        /// <remarks>
+        ///   If a connection is closed due to being idle, the Event Hubs clients will automatically
```

Comment:
I believe this needs to be changed to "Service Bus clients"...

Reply:
<img src="https://media4.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif"/>

Reply:
I'm just impressed at how quickly you got this fix in. Thanks again!

Reply:
Thanks for the heads-up.   ##35958 should fix that.